fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlanner.java

```diff
@@ -783,7 +786,7 @@ private PlanContext getPlanContext(@Nonnull RecordQuery query) {
             if (query.getRecordTypes().isEmpty()) { // ALL_TYPES
                 commonPrimaryKey = RecordMetaData.commonPrimaryKey(metaData.getRecordTypes().values());
             } else {
-                final List<RecordType> recordTypes = query.getRecordTypes().stream().map(metaData::getRecordType).collect(Collectors.toList());
+                final List<RecordType> recordTypes = query.getRecordTypes().stream().map(metaData::getIndexableRecordType).collect(Collectors.toList());
```

Comment:
`getQueryableRecordType`, again.

Reply:
changed all those new refs to that method to use the new method